README.md

```diff
@@ -2,6 +2,7 @@
 
 Cartopy
 -------
+[![Build Status](https://api.travis-ci.org/repositories/SciTools/cartopy.svg?branch=master)](http://travis-ci.org/SciTools/cartopy/branches)
```

Comment:
the http just redirects to https at travis, so I don't think this matters.

This is needed so that we can respin tests, which are currently failing due to this test matrix change.

i'm going to merge this

it's easy to alter the URI in the readme as required

@QuLogic this is the same as on the Iris readme:
https://raw.githubusercontent.com/SciTools/iris/master/README.md
which this pattern was taken from.
would you prefer that the URIs were consistent and using https on both of these repositories?

mark